posts/when_to_fold_em.md

```diff
@@ -0,0 +1,48 @@
+
+> We’ve learned some things about what we’re trying to tackle here. Let’s give it another couple of months and see if we can make any progress.
+
+Of course, a month later, nothing had changed - it was the same team, with the same tools, and the same level of support. But we didn't look at how to make it better. Instead, we talked about "not giving up":
+
+> We’ve already sunk so much money into this. We can’t throw that away now. Let’s keep fighting!
+
+This is the **sunk cost** fallacy in action - treating what you’ve already lost as a factor in the decision on what to do next.
+
+Our brains were compiled in caves, and some of the software is very out-of-date. Today, we’re going to do some bug-fixing.
+
+
+## Let the project die - kill it, if you have to
+
+But what is the bug? Aren’t you right to care about wasting money?
+
+Sure, waste is bad. But if you’re looking at it in the past, it’s already wasted. Since the past loss is happening no matter what, *only look at the future results* if you want to make the right decision.
+
+Let’s say this half-finished project of ours has cost £100,000 so far, and the cost of finishing it will be another £50,000. When complete, it’ll reward us with £150,000 of business value.
+
+Alternately, we’ve got another opportunity that costs £20,000, but makes us £100,000.
+
+**Option A**: Spend £150,000 total, get £150,000
+**Option B**: Spend £120,000 total, get £100,000
+
+And that makes option A look appealing - at least you make your money back, right? Option B looks like it’s a loss! But you can’t change the past, so your current decision is actually...
+
+**Option A**: Spend £50,000, get £150,000 (x3 return)
+**Option B**: Spend £20,000, get £100,000 (x5 return)
+
+If you wanted to maximise return on investment - and you don’t have a time machine, **option B is the rational choice**.
+
+In real life, decisions aren’t this clear cut - outcomes can’t be exactly predicted - but the same rule holds. Decide based on the costs and benefits from now on, because the past cost is already sunk, in both cases.
+
+## Lazy load those decisions
+
+This is all a more specific application of one of the key parts of iterative development - *we’ll learn as we go, so we’ll make better decisions later on*. It’s important to stop, breathe and reflect, even when you’re in the middle of work, or you’ll miss opportunities to make the right decisions.
+
+But once you’ve learnt this lesson, there’s still some work to do. Whether you’re an executive trying to encourage this in your culture, or a developer trying to make the right decisions, you need to understand the *context*.
+
+Making the rational choice isn't the only thing that motivates people. Don’t fall into the trap of explaining why abandoning a sunk cost might be the best decision, whilst neglecting the emotional attachment your colleagues may have to their work.
```

Comment:
"Remember that making the rational choice isn't the only thing that motivates people." 
sets the scene for the following sentence where you give direct advice

Reply:
Good idea, adding this (with sliiightly different wording)